### ISSUE TITLE
Cloud-init.md documentation  does not reflect alle the current changes

### ISSUE THREAD

**Issue by TimoGoetze** [Telmate/terraform-provider-proxmox#972]
hi together,

I am quite new to terraform proxmox driver. I could not  get it to run properly by the documentation and all the cloud-init issues around 3.0.1rc1 or lasrt 2.9 confused me. 

could anybody with knowledge please spend 5 minutes and update the 

https://github.com/Telmate/terraform-provider-proxmox/blob/master/docs/guides/cloud_init.md

Thank you so much !

Cheers, 

Timo

**Comment by vlkyrylenko** [comment]
## First of all, I appreciate the work you've done maintaining this tf provider.
I'm also new to proxmox and I had to follow  a couple of outdated guides and I must admit that figuring out how the new blocks work is a really frustrating experience.
I'd really appreciate if someone could offer some guidance and point me in the right direction.
My main struggle was working with the new `disks` and `network` blocks.
If someone could update the example posted in this repository, I'm sure it will inevitably draw more people and get them up to speed in a matter of minutes.
Thanks again!

**Comment by tuxthepenguin84** [comment]
Here's an example I've used for disks:
https://github.com/Telmate/terraform-provider-proxmox/blob/master/examples/pxe_example.tf#L52

**Comment by vlkyrylenko** [comment]
Yes, that's a great starting point. I've also managed to import the state of previously created resources.
So given the [pxe example](https://github.com/Telmate/terraform-provider-proxmox/blob/master/examples/pxe_example.tf#L52) and my own TF state I was able to recreate my cluster with Terraform.
Perhaps I can contribute and create a PR with another more beginner-friendly example once I clean it my code and add more comments.

Here's the TF command I used to export my TF state into an already existing module where <vm_name> is the name of the actual TF resource defined in that module.

```terraform import 'proxmox_vm_qemu.<vm_name>' <host_name>/proxmox_vm_qemu/<vm_id>```
